### PR TITLE
Admin create & destroy course

### PR DESCRIPTION
**Pull request by QuangVinh93**

### CODE REVIEW

**Comment by hitshootnum1** [comment]
ok

**Comment by HUSTToanLH** [comment]
ok

File: app/controllers/admin/courses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+class Admin::CoursesController < ApplicationController
+  before_action :load_all_courses
+  before_action :load_new_course
```
Comment (dieunb) [inline app/controllers/admin/courses_controller.rb]: should define this use for actions

File: app/controllers/admin/courses_controller.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+class Admin::CoursesController < ApplicationController
+  before_action :load_all_courses
```
Comment (dieunb) [inline app/controllers/admin/courses_controller.rb]: should define this use for what actions

File: app/controllers/admin/courses_controller.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  before_action :load_new_course
+  before_action :load_all_subjects
```
Comment (dieunb) [inline app/controllers/admin/courses_controller.rb]: should define this use for what actions

**Pull request merged**